docs/_advanced/middleware_listener.md

```diff
@@ -8,15 +8,35 @@ order: 5
 <div class="section-content">
 Listener middleware is used for logic across many listener functions (but usually not all of them). They are added as arguments before the listener function in one of the built-in methods. You can add any number of listener middleware before the listener function.
 
-There’s a collection of built-in listener middleware that you can use like `subtype()` for filtering on a message subtype and `directMention()` which filters out any message that doesn’t directly @-mention your bot.
+There’s a collection of built-in listener middleware that you can use like `subtype()` for filtering on a message subtype and `directMention()` which filters out any message that doesn’t directly @-mention your bot "at the beginning of the message". 
```

Comment:
I'm not sure if the quotes around the "at the beginning of the message are needed.

Reply:
removed quotes, that was a placeholder to mark change and should not have been there